### PR TITLE
experiment/defects (research for feature/SOF-7360)

### PR DESCRIPTION
**Pull request by VsevolodX**
- feat: add defect creation nb

### CODE REVIEW

File: other/materials_designer/add_vacancy_defect.ipynb [hunk 1]
```diff
@@ -0,0 +1,220 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/add_vacancy_defect.ipynb:1]: <p>Create point defects in a bulk material should be the name of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: other/materials_designer/create_defect.ipynb [hunk 2]
```diff
@@ -0,0 +1,495 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_defect.ipynb:1]: <p>This notebook should not exist - you can keep it in a separate backup branch, but not in something we will merge</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: other/materials_designer/Introduction.ipynb [hunk 3]
```diff
@@ -25,6 +25,10 @@
     "\n",
```
Comment (timurbazhirov) [inline other/materials_designer/Introduction.ipynb:25]: <p>Remove "bulk material" from 1.4 and its explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: other/materials_designer/create_point_defect_in_bulk.ipynb [hunk 4]
```diff
@@ -0,0 +1,273 @@
+{
```
Comment (timurbazhirov) [inline other/materials_designer/create_point_defect_in_bulk.ipynb:1]: <p>Prepare defect configurations as title</p><p> </p><p><code>defect_configuration_with_position</code></p><p><code>defect_configuration_with_site_id</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (timurbazhirov) [inline other/materials_designer/create_point_defect_in_bulk.ipynb:1]: <p>materials_with_defect_at_site_id</p><p>materials_with_defect_at_position</p><p> </p><p>and titles accordingly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

File: other/materials_designer/Introduction.ipynb (hunk 3 above)
Comment (VsevolodX) [reply other/materials_designer/Introduction.ipynb:25]: <p>Ok</p>

File: other/materials_designer/create_point_defect_in_bulk.ipynb (hunk 4 above)
Comment (VsevolodX) [reply other/materials_designer/create_point_defect_in_bulk.ipynb]: <p>OK</p>

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**